test/system/130-kill.bats

```diff
@@ -6,12 +6,23 @@
 load helpers
 
 @test "podman kill - test signal handling in containers" {
+    # podman-remote and crun interact poorly in f31: crun seems to gobble up
+    # some signals.
+    # Workaround: run 'env --default-signal sh' instead of just 'sh' in
+    # the container. Since env on our regular alpine image doesn't support
+    # that flag, we need to pull fedora-minimal. See:
+    #    https://github.com/containers/libpod/issues/5004
+    # FIXME: remove this kludge once we get rid of podman-remote
+    local _image=$IMAGE
+    local _sh_cmd="sh"
+    if is_remote; then
+        _image=quay.io/libpod/fedora-minimal:latest
```

Comment:
FWIW and/or maybe a helpful addition to this comment.  This image is built automatically from `./contrib/fedora-minimal/` (since the fedoraproject registry is so often flaky).

Reply:
Thanks - I think the `quay.io` is enough to clue someone in that this is a special image; but in this test my goal is to minimize all pulls, even from quay. Every pull is a high-risk chance for a flake.

Reply:
Yes I agree with the methodology.

Actually, something that occurred to me:  When the ginkgo tests run, we cache image tarballs in `/tmp` with some very stable/well-known filenames.  If those files exist, pulling could be completely avoided...just import them directly.